### DISCUSSION TITLE
HELP - Endless loading screen

### DISCUSSION THREAD

**Discussion by DavidJMChapman** [CitiesSkylinesMods/TMPE#1639] [Q&A] [answered]
Hi, my name is David and I have downloaded the newest version of cities skyline from Steamunlocked and I was wondering why the mod is not working, could anyone guide me through how to download and enable the mod, I have gone into C:\Users\david\AppData\Local\Colossal Order\Cities_Skylines\Addons\Mods and created a folder name TrafficManager and another one for Citiesharmony, but it is still not Working HELP plz

**Comment by krzychu124** [comment, marked as answer]
https://github.com/CitiesSkylinesMods/TMPE/wiki/Installation
